File: freecodecamp/bonfire/12-shasher-flick.js
function slasher(arr, howMany) {
  arr.splice(0, howMany);
  return arr;
}

console.log(slasher([1,2,3], 2));
console.log(slasher([1,2,3], 0));
console.log(slasher([1,2,3], 9));
console.log(slasher([1,2,3], 4));

File: freecodecamp/bonfire/12-shasher-flick.rb
def slasher(array, how_many)
  array.drop(how_many)
end

p slasher([1,2,3], 2)
p slasher([1,2,3], 0)
p slasher([1,2,3], 9)
p slasher([1,2,3], 4)
